check/pytest-changed-files

```diff
@@ -39,9 +63,9 @@ changed=$(git diff --name-only "$1" \
 )
 num_changed=$(echo -e "${changed}" | wc -w)
 
+# Run it.
 echo "Found ${num_changed} differing files with associated tests."
 if [ -z "${changed}" ]; then
     exit 0
 fi
-
-pytest "${@:2}" ${changed}
+pytest "${rest}" ${changed}
```

Comment:
The quotes need to be removed so $rest can be expanded.

Reply:
Fixed.